dune-dev-deps.opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
```

Comment:
I think that having this new opam file will cause `dune-release` to submit it to opam-repository at release time. How about replacing this dummy package by an explicit list in the Makefile?

Reply:
I was vaguely worried about that at one point, but forgot to give it a try to see. Moving it into the Makefile will accomplish the same end, and is arguably more correct :+1:  Will do!

Reply:
Fixed, thanks :)